CONTRIBUTING.md

```diff
@@ -29,92 +29,40 @@ For a brief history and our ethos/design philosophy around the Amplify project p
 
 ## Getting Started
 
-To get started with the Amplify Framework for iOS, first make sure you have the latest version of Xcode installed as well as cocoapods. You can install cocoapods by simply running:
-
-```
-sudo gem install cocoapods
-```
+To get started with the Amplify Library for Swift, first make sure you have the latest version of Xcode installed.
```

Comment:
should we add the min XCode version supported?

Reply:
good call.  done.